HesoyamHospital/GraphicEditor/SearchService.cs

```diff
@@ -11,73 +11,74 @@ public SearchService()
         {
-            List<MapLocation> map_locations = new List<MapLocation>();
-            List<MapLocation> map_location1 = Find_in_file("hospital1groundfloor.txt", name);
-            List<MapLocation> map_location2 = Find_in_file("hospital1firstfloor.txt", name);
-            List<MapLocation> map_location3 = Find_in_file("hospital2groundfloor.txt", name);
-            List<MapLocation> map_location4 = Find_in_file("hospital2firstfloor.txt", name);
-            List<MapLocation> map_location5 = Find_in_file("hospital2secondfloor.txt", name);
+            List<MapLocation> mapLocations = new List<MapLocation>();
+            List<MapLocation> mapLocation1 = Find_in_file("hospital1groundfloor.txt", name);
+            List<MapLocation> mapLocation2 = Find_in_file("hospital1firstfloor.txt", name);
+            List<MapLocation> mapLocation3 = Find_in_file("hospital2groundfloor.txt", name);
+            List<MapLocation> mapLocation4 = Find_in_file("hospital2firstfloor.txt", name);
+            List<MapLocation> mapLocation5 = Find_in_file("hospital2secondfloor.txt", name);
 
-            map_locations.AddRange(map_location1);
+            mapLocations.AddRange(mapLocation1);
             
-            map_locations.AddRange(map_location2);
+            mapLocations.AddRange(mapLocation2);
 
-            map_locations.AddRange(map_location3);
+            mapLocations.AddRange(mapLocation3);
             
-            map_locations.AddRange(map_location4);
+            mapLocations.AddRange(mapLocation4);
             
-            map_locations.AddRange(map_location5);
+            mapLocations.AddRange(mapLocation5);
 
-            return map_locations;
+            return mapLocations;
         }
 
         public List<MapLocation> Find_in_file(string path, string name)
         {
-            MapLocation map_location = null ;
-            List<MapLocation> map_locations = new List<MapLocation>();
+            List<String> names = new List<string>();
+            MapLocation location = null ;
+            List<MapLocation> locations = new List<MapLocation>();
             GraphicRepository graphic_repository = new GraphicRepository();
             List<GraphicalObject> graphical_objects = graphic_repository.ReadFromFile(path);
 
             foreach (GraphicalObject obj in graphical_objects) {
                 if (obj.Name.Contains(name)) 
                 {
                     string full_name = obj.Name;
-                    map_location = Get_Location_By_Filename(path, full_name);
-                    map_locations.Add(map_location);
+                    location = Get_Location_By_Filename(path, full_name);
+                    locations.Add(location);
                     
                 }
             }
 
-            return map_locations;
+            return locations;
         
         }
 
         public MapLocation Get_Location_By_Filename(string path, string name) {
-            MapLocation map_location = new MapLocation();
```

Comment:
Wouldn't it be better if you had all this data stored somewhere?
What would happen if we had hospital 3?
As from my understanding, another conditional statement would have to be placed and code re-compiled.
That's not the smartest solution, therefore please take a look into that. Let's try to make this as much dynamic as it can be.